app/models/manageiq/providers/kubernetes/container_manager_mixin.rb

```diff
@@ -102,19 +102,52 @@ def default_authentication_type
     :bearer
   end
 
-  def scan_job_create(entity_class, entity_id)
+  def scan_job_create(entity)
+    check_policy_prevent(:request_containerimage_scan, :raw_scan_job_create, entity)
```

Comment:
The change from class + id to entity, while cleaner, now means the whole entity will need to be serialized on the queue (IIUC).  I believe serializations of AR objects on the queue are being hunted down.